tutorials/detecting-changes-in-sentinel-1-imagery-pt-3/tutorial.ipynb

```diff
@@ -0,0 +1 @@
Run the algorithm with median filter and at 1% significance.\n","result = ee.Dictionary(change_maps(im_list, median=True, alpha=0.01))\n","# Extract the change maps and export to assets.\n","cmap = ee.Image(result.get('cmap'))\n","smap = ee.Image(result.get('smap'))\n","fmap = ee.Image(result.get('fmap'))\n","bmap = ee.Image(result.get('bmap'))\n","cmaps = ee.Image.cat(cmap,smap,fmap,bmap).rename(['cmap','smap','fmap']+timestamplist[1:])\n","\n","#         CHANGE THIS TO THE OWN USERNAME/PATH.\n","assetId =  'users/YOUR_USER_NAME/cmaps'\n","\n","assexport = ee.batch.Export.image.toAsset(cmaps,\n","                    description = 'assetExportTask', \n","                    assetId = assetId, scale=10, maxPixels=1e9)      \n","# Uncomment this for exporting.\n","#assexport.start()"],"execution_count":null,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"8eBeTB6sQMbd"},"source":["The asset  _cmaps_ is shared so we can all access it:"]},{"cell_type":"code","metadata":{"id":"qseBmVOH7xPg"},"source":["cmaps = ee.Image('projects/earthengine-community/tutorials/detecting-changes-in-sentinel-1-imagery-pt-3/cmaps')\n","cmaps = cmaps.updateMask(cmaps.gt(0))\n","\n","location = aoi.centroid().coordinates().getInfo()[::-1]\n","palette = ['black','blue','cyan','yellow','red']\n","palette1 = ['black','red','cyan','yellow']\n","mp = folium.Map(location=location, zoom_start=13, width=1000)\n","\n","mp.add_ee_layer(cmaps.select('T20191107'), {'min': 0,'max': 3, 'palette': palette1}, 'T20191107')\n","mp.add_ee_layer(cmaps.select('T20191113'), {'min': 0,'max': 3, 'palette': palette1}, 'T20191113')\n","mp.add_ee_layer(cmaps.select('T20191119'), {'min': 0,'max': 3, 'palette': palette1}, 'T20191119')\n","mp.add_ee_layer(cmaps.select('T20191125'), {'min': 0,'max': 3, 'palette': palette1}, 'T20191125')\n","mp.add_ee_layer(cmaps.select('T20191201'), {'min': 0,'max': 3, 'palette': palette1}, 'T20191201')\n","mp.add_ee_layer(cmaps.select('T20191207'), {'min': 0,'max': 3, 'palette': palette1}, 'T20191207')\n","\n","mp.add_child(folium.LayerControl())"],"execution_count":null,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"WFuLna6zQ_zt"},"source":["Now interpretation is somewhat easier. The negative definite (cyan) changes which appear between Nov. 7 and Nov. 13 correspond to decreases in intensity of _VV_ and _VH_ reflectance and are due to wide-spread flooding. The positive definite changes (red), which gradually overlay the flooded areas in subsequent intervals, correspond to receding flood waters.\n"]},{"cell_type":"markdown","metadata":{"id":"-WhBsXEjsZCU"},"source":["### Outlook\n","\n","Without reliable ground truth we can't really claim that change maps of the kind we have just generated will be helpful for flood damage assessment or control, but their potential usefulness is quite obvious. In the next and final part of the Tutorial we will have a look at some more (possible) applications of sequential change detection with SAR imagery using GEE."]}]}
```

Comment:
<p>Please remove extra "()" around assignments of im_coll, e.g.,:</p><pre class="ql-syntax" spellcheck="false">im_coll = ee.ImageCollection('COPERNICUS/S1_GRD_FLOAT')&nbsp;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;.filterBounds(aoi)&nbsp;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;.filterDate(ee.Date('2019-09-01'),ee.Date('2020-01-31'))&nbsp;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;.filter(ee.Filter.eq('orbitProperties_pass', 'DESCENDING'))&nbsp;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;.filter(ee.Filter.eq('relativeOrbitNumber_start', 154))&nbsp;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;.sort('system:time_start')
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
@gino-m The parentheses  are needed to avoid using " \ " for line continuation. Suggested by Justin.

Reply:
For reference, the Google Python style guide states:

> Do not use backslash line continuation except for with statements requiring three or more context managers.

> Make use of Python’s implicit line joining inside parentheses, brackets and braces. If necessary, you can add an extra pair of parentheses around an expression.

https://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
<p>Sorry, I didn't realize hanging indents weren't supported. SGTM!</p>